hermes-common/src/main/java/pl/allegro/tech/hermes/common/kafka/KafkaZookeeperPaths.java

```diff
@@ -0,0 +1,38 @@
+package pl.allegro.tech.hermes.common.kafka;
+
+import com.google.common.base.Joiner;
+
+public final class KafkaZookeeperPaths {
+
+    private KafkaZookeeperPaths() {
+    }
+
+    public static String offsetsPath(ConsumerGroupId groupId, KafkaTopicName topicName) {
+        return join(consumerGroupPath(groupId), "offsets", topicName.asString());
```

Comment:
Maybe we can reuse somehow kafka.utils.ZkUtils.scala which contains paths that are needed here.